components/totango/sources/new-account/new-account.mjs

```diff
@@ -0,0 +1,59 @@
+import totango from "../../totango.app.mjs";
+
+export default {
+  name: "New Account",
+  version: "0.0.1",
+  key: "totango-new-account",
+  description: "Emit new event for each created account",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    totango,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      static: {
+        intervalSeconds: 15 * 60, // 15 minutes
+      },
+    },
+  },
+  methods: {
+    emitEvent(data) {
+      this.$emit(data, {
+        id: data.name,
+        summary: `New account with name ${data.name}`,
+        ts: new Date(),
+      });
+    },
+  },
+  hooks: {
+    async deploy() {
+      const accounts = await this.totango.searchAccounts({
+        data: {
+          "query": "{\"terms\":[],\"count\":10,\"offset\":0,\"fields\":[],\"scope\":\"all\"}",
+        },
+      });
+
+      accounts.reverse().forEach(this.emitEvent);
+    },
+  },
+  async run() {
+    let page = 0;
+
+    while (page >= 0) {
+      const accounts = await this.totango.searchAccounts({
```

Comment:
Would not it be returning all related accounts when timer ticks? Can we record something to `db` and check new accounts according to that?

Reply:
I don't know what is the best way to work with that, we can save in db and compare, or just emit and the core will compare.